flake.nix

```diff
@@ -243,16 +255,6 @@
             ];
             auditable = false; # Avoid cargo-auditable failures.
             doCheck = false; # Disable test as it requires network access.
-
-            CARGO_BUILD_TARGET = rustTarget;
```

Comment:
Had to remove this so the build succeeds, what were the intentions and is it really needed? @link2xt